### PR TITLE
idm introduction and reference

### PR DESCRIPTION
**Pull request by pb82**
Intro and terminology for the IDM section.

### CODE REVIEW

File: modules/ROOT/pages/keycloak/index.adoc [hunk 1]
```diff
@@ -6,7 +6,16 @@ include::{partialsdir}/attributes.adoc[]
 [id='{context}_con_introduction-to-keycloak']
 = About {keycloak-service}
 
+The {keycloak-service} service allows you to add authentication and authorization to your mobile app.
+
+* Secure your mobile app using the industry standard OpenID Connect protocol
+* Add access control to your app based on user’s group membership
+* Easily implement SSO, multi factor authentication and Social Login support
+* Back end support for identity brokering and user federation
```
Comment (josemigallas) [inline modules/ROOT/pages/keycloak/index.adoc]: I think Backend goes in 1 word?

**Comment by jstaffor** [comment]
@finp - can you implement the tagging system that you have created in https://github.com/aerogear/mobile-docs/pull/59 for this?

File: modules/ROOT/pages/keycloak/index.adoc (hunk 1 above)
Comment (finp) [reply modules/ROOT/pages/keycloak/index.adoc]: back-end is the correct term from IBM Style guide

**Comment by pb82** [comment]
@finp Ok. What should we put into `index.adoc` for the individual services?

**Pull request merged**